internal/controller/dispatch_logic.go

```diff
@@ -204,13 +204,38 @@ func (r *Dispatcher) selectForDispatch(ctx context.Context) ([]*mcadv1beta1.AppW
 	// return ordered slice of AppWrappers that fit (may be empty)
 	for _, appWrapper := range queue {
 		request := aggregateRequests(appWrapper)
+		// get resourceQuota in AppWrapper namespace, if any
+		resourceQuotas := &v1.ResourceQuotaList{}
+		namespace := appWrapper.GetNamespace()
+		if err := r.List(ctx, resourceQuotas, client.UnsafeDisableDeepCopy,
+			&client.ListOptions{Namespace: namespace}); err != nil {
+			return nil, err
+		}
+		quotaFits := true
+		var appWrapperAskWeights *WeightsPair
+		insufficientResources := []v1.ResourceName{}
+		if len(resourceQuotas.Items) > 0 {
+			appWrapperAskWeights = getWeightsPairForAppWrapper(appWrapper)
+			// TODO: relax assumption that only one resourceQuota per nameSpace
```

Comment:
Please open an issue (or comment on an existing issue) to document this limitation and others.

Reply:
added to issue #44 .